src/modules/organizations/containers/links-container.jsx

```diff
@@ -45,31 +45,35 @@ class LinksContainer extends React.Component {
       <div>
         <h5>Links</h5>
         <div>
-          <span className="form__label" htmlFor="external urls">
+          <label className="form form__label" htmlFor="external urls">
             External Links
-            <ExternalLinksEditor
-              id="external"
-              name="external"
-              onChange={this.handleUrlsChange}
-              urls={this.state.urls}
-            />
-          </span>
+            <div className="form__input--top-margin">
+              <ExternalLinksEditor
```

Comment:
We should add a className` prop to this component, if it doesn't have one already, to avoid the wrapper `div`.